Update DNS records in pritesh.json

epritesh opened the pull request:
<!--
YOU MUST FILL OUT THIS TEMPLATE ENTIRELY FOR YOUR PR TO BE ACCEPTED, NONE OF IT IS OPTIONAL UNLESS SPECIFIED.
-->

# Requirements
- [x] I have **read** and **agreed** to the [Terms of Service](https://is-a.dev/terms).
- [x] My file is following the [domain structure](https://docs.is-a.dev/domain-structure/).
- [x] My website is **reachable** and **completed**.
- [x] My website is **software development** related.
- [x] My website is not for commercial use.
- [x] I have provided sufficient contact information in the `owner` key.
- [x] I have provided a preview of my website below.

# Website Preview
- Live site: https://personal-website-ochre-mu-51.vercel.app/
- Contact: Discord @epritesh (display name Nazzu)
- Note: Adding `_vercel` TXT record requested for domain verification.

epritesh committed:
Update DNS records in pritesh.json

epritesh commented:
@STICKnoLOGIC

epritesh committed:
Merge branch 'main' into patch-6

STICKnoLOGIC commented on domains/pritesh.json:
```diff
@@ -4,6 +4,9 @@
   },
   "records": {
-    "CNAME": "personal-website-ochre-mu-51.vercel.app"
+    "A": ["76.76.21.21"],
+    "TXT": [
+      "_vercel=vc-domain-verify=pritesh.is-a.dev,a297d9122cd5f6fad482"
+    ]
```
check the step 5 -> https://docs.is-a.dev/guides/vercel/

you need to also create _vercel.pritesh.json and put the TXT records there
```suggestion
    "A": ["76.76.21.21"]
```

epritesh committed:
Merge branch 'main' into patch-6

epritesh commented:
Thanks! I’ve updated domains/pritesh.json to include only the A record and added domains/_vercel.pritesh.json with the verification TXT as in the guide. Let me know if anything else is needed.

epritesh committed:
Update domains/pritesh.json

Co-authored-by: John Aerial J. Azcune <65322242+STICKnoLOGIC@users.noreply.github.com>

epritesh committed:
Add Vercel verification file for pritesh.is-a.dev

epritesh committed:
Merge branch 'main' into patch-6

epritesh committed:
Move Vercel verification file to correct path

epritesh committed:
Delete domains/_vercel.pritesh.json

epritesh committed:
Add Vercel configuration for domain verification

epritesh committed:
Add Vercel configuration for domain verification

epritesh committed:
Add Vercel verification TXT for pritesh.is-a.dev

epritesh committed:
Update _vercel.pritesh.json

epritesh committed:
Update _vercel.pritesh.json

STICKnoLOGIC commented on domains/is-a.dev/_vercel.pritesh.json:
remove this file T_T

epritesh committed:
Delete domains/is-a.dev/_vercel.pritesh.json

epritesh committed:
Merge branch 'is-a-dev:main' into patch-6

epritesh reviewed (commented):
Fixed domain config

epritesh committed:
fix: keep TXT-only in _vercel.pritesh.json (A record stays in pritesh.json)

epritesh committed:
fix: _vercel TXT record format (string, no _vercel= prefix)

STICKnoLOGIC reviewed (approved):
lgtm
